data.yml

```diff
@@ -38,6 +38,29 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BadRequestError'
+              examples:
+                      Bad Request:
+                       value:
+                        type: list
+                        correlationId: ac5ah5i
+                        data:
+                          - type: error
+                            title: Parameter not valid.
+                            code: parameter-not-valid
+                            detail: ID value is not valid.
+                            source:
+                              parameter: id
+                      Invalid Filters:
+                        value:
+                          type: list
+                          correlationId: ac5ah55
+                          data:
+                            - type: error
+                              code: parameter-not-valid
+                              title: Parameter value is not valid
+                              detail: The provided filter parameter is in invalid format or unsupported
+                              source:
+                                parameter: filter
         403:
           description: Error that access is forbidden and tied to the application
```

Comment:
Is this read as 

`Error that access is forbidden and tied to the application logic, such as insufficient rights to a resource.`

or 

`Error. That access is forbidden and tied to the application logic, such as insufficient rights to a resource.`

??

Reply:
Interesting. We can add the dot here.